sdk/storage/azure-storage-blob/CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## 12.3.3 (Unreleased)
 
+**New features**
+
+- Added support for object replication properties for download response and get properties.
```

Comment:
is this updated?

Reply:
Thanks for catching this! I have an open pr to update all changelog files https://github.com/Azure/azure-sdk-for-python/pull/12261